### PR TITLE
Add man pages for zstdgrep and zstdless

### PR DESCRIPTION
**Pull request by samrussell**
zgrep and zless come with man pages, this adds the same to zstdgrep and zstdless

The zstdgrep man page closely mirrors the zgrep man page; the zstdless is a little... less

### CODE REVIEW

File: programs/zstdless.1.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+zstdless(1) -- view zstandard-compressed files
+============================================================================
+
+SYNOPSIS
+--------
+
+`zless` [*flags*] [_file_ ...]
```
Comment (Cyan4973) [inline programs/zstdless.1.md]: should it be `zstdless` ?

File: programs/zstdgrep.1.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+zstdgrep(1) -- print lines matching a pattern in zstandard-compressed files
+============================================================================
+
+SYNOPSIS
+--------
+
+`zgrep` [*grep-flags*] [--] _pattern_ [_files_ ...]
```
Comment (Cyan4973) [inline programs/zstdgrep.1.md]: should it be `zstdgrep` ?

**Comment by Cyan4973** [comment]
Thanks @samrussell , it's a welcomed doc addition !

Just a couple of comments, as I suspect some left-over from `zgrep`/`zless` documentation.

**Comment by samrussell** [comment]
Thanks @Cyan4973, got a bit wordblind at the end, should be no more zgrep/zless references

**Comment by Cyan4973** [comment]
Thanks for the patch @samrussell ! 
Will be part of next release !

**Pull request merged**